app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    post_id = params.require('post_id')
```

Comment:
[OPTIONAL] As stated above it is better to use strong parameters in a private method